autogluon/utils/tabular/ml/models/abstract/abstract_model.py

```diff
@@ -314,7 +314,7 @@ def convert_to_compressed_template(self):
         template.path = template.create_contexts(self.path + template.name + os.path.sep)
         return template
 
-    def hyperparameter_tune(self, X_train, X_test, Y_train, Y_test, scheduler_options=None, **kwargs):
+    def hyperparameter_tune(self, X_train, X_test, Y_train, Y_test, scheduler_options, **kwargs):
```

Comment:
should every model's hyperparameter_tune require scheduler_options, when some don't even utilize a scheduler?

Reply:
For the ones that don't they can have it default to None, but for AbstractModel's version it is required:

```
scheduler_func, scheduler_options = scheduler_options  # Unpack tuple
        if scheduler_func is None or scheduler_options is None:
            raise ValueError("scheduler_func and scheduler_options cannot be None for hyperparameter tuning")
```